citi-onboarding/PubliEasy
# Feature/about us

## What I did

- NavBar
   - Logo na esquerda
   - Lista com os 4 links (sendo cada um um item da lista) e o contato destacado com uma borda com linear gradient (dentro de uma div)
   - Navbar fixa independente da rolagem da pagina
- About Us
   - Div com o espaço geral do sobre nos
   - Div com a parte da foto como background e dentro dela a div dos textos e o botão de rolagem (botao no centro e embaixo)
   - Na div dos textos, outra tag para a ultima palavra diferenciar a cor, texto alinhado na direita 
   - Paragrafo com o Lorem ipsum, com a ultima frase com outra tag para diferenciar a espessura 


OBS.: eu me confundi e fiz a maioria das coisas tanto da nav bar quanto do aboutus nessa branch, ai só vou enviar essa mesmo
## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/aboutUs`
   1. `git pull origin feature/aboutUs`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 4h30min

**Resolves:** #<issue number>
